packages/erc3k/contracts/IERC3000.sol

```diff
@@ -8,20 +8,25 @@ pragma experimental ABIEncoderV2;
 import "./ERC3000Data.sol";
 import "./IERC3000Executor.sol";
 
+// Note: why not defining functions as external?
 abstract contract IERC3000 {
     function schedule(ERC3000Data.Container memory container) virtual public returns (bytes32 actionHash);
+    // Note: why logging just the collateral? wouldn't be better to log a Config hash or identifier?
     event Scheduled(bytes32 indexed containerHash, ERC3000Data.Payload payload, ERC3000Data.Collateral collateral);
 
+    // Note: why enforcing this in the interface? it could be confused with the executor interface
```

Comment:
I don't understand. I think we could rename this function so it is not as confusing, but I do believe that this needs to be part of the standard since it is part of the core flow. Rename suggestions?

Reply:
I also struggle with the naming of this function and the name collision it has with the `ERC3000Executor`. What you think about `enact` and `Enacted` as names?